site/source/docs/compiling/Travis.rst

```diff
@@ -0,0 +1,72 @@
+.. _Travis:
+
+==============================
+Building projects on Travis CI
+==============================
+
+`Travis CI <https://travis-ci.org/>`_ is a popular continuous integration service which offers free plans for open source projects. Thanks to a `Docker image by trzeci <https://hub.docker.com/r/trzeci/emscripten/>`_ installing emscripten in Travis CI is essentially a one line task.
+
+An example .travis.yml
+======================
+
+.. code-block:: yaml
+
+    notifications:
+      email: false
+
+    language: node_js
+    node_js:
+      - node
+
+    sudo: required
+
+    services:
+      - docker
+
+    before_install:
+      - docker run -dit --name emscripten -v $(pwd):/src trzeci/emscripten:sdk-tag-1.37.22-64bit bash
+
+    script:
+      - docker exec -it emscripten make helloworld.js
+      - make test
+
+Let's break it down:
+
+.. code-block:: yaml
+
+    notifications:
+      email: false
+
+    language: node_js
+    node_js:
+      - node
+
+    sudo: required
+
+    services:
+      - docker
+
+These lines set up the basic settings for the Travis container. Most people do not want email notifications, but feel free to leave out those lines if you do.
+
+``language: node_js`` and ``node_js: - node`` tell Travis we are a Node.js project, and that we want the latest stable Node release.
+
+``sudo: required`` and ``services: - docker`` are required to enable Docker in the Travis container.
+
+.. code-block:: yaml
+
+    before_install:
+      - docker run -dit --name emscripten -v $(pwd):/src trzeci/emscripten:sdk-tag-1.37.22-64bit bash
+
+In the before_install stage we download the Docker image, create a container with that image, and then give it the name ``emscripten``. The ``-dit`` options tell Docker that we want the container to run *bash* in the background.
+
+This Docker image contains everything emscripten needs to run, as well as several additional build tools such as *make* and *cmake*. If you do not need them you can use the `emscripten-slim image <https://hub.docker.com/r/trzeci/emscripten-slim/>`_ instead, which excludes them and will be downloaded and installed slightly quicker. The emscripten versions available are listed at `the Docker Hub <https://hub.docker.com/r/trzeci/emscripten/tags/>`_. Neither image includes tools such as *curl*, so if you need to download any files, do so as a normal Travis command rather than a Docker command.
```

Comment:
Just a question, the full image contains `wget`. I can add `curl` also. If you see any package what's common in usage, please let me know.

Reply:
Oh, I hadn't made the connection between your usernames. This is fantastic btw!

I hadn't thought of using wget, and wasn't trying to change my Makefile much. I don't think details like really need to be laid out in this doc. I also don't think the Docker image needs curl - I don't need it at least!

Reply:
No worries, I'll be glad to make the image as universal as it is possible.

Reply:
Missing `curl` was fixed in `incoming` build, and it will be a part of 1.37.23: https://github.com/asRIA/emscripten-docker/issues/14 
In this context this extra note could be removed perhaps to don't mislead readers. :)

Reply:
@asRIA Already removed it earlier today :)